demo/toast.py

```diff
@@ -0,0 +1,182 @@
+"""Simple toast component that is similar to Angular Component Snackbar."""
+
+import time
+from typing import Callable, Literal
+
+import mesop as me
+
+
+@me.stateclass
+class State:
+  is_visible: bool = False
+  duration: int = 0
+  horizontal_position: str = "start"
```

Comment:
another nitpicky comment: I'd pick as default

horizontal = "center"
vertical = "end"

Although the user can select it from the dropdown, I think it's nice to use these values as the default (and you already use it as the default in the function parameters)

Reply:
Sure. No problem. Updated.